patches/chrome-browser-profiles-profile_io_data.cc.patch

```diff
@@ -1,14 +1,12 @@
 diff --git a/chrome/browser/profiles/profile_io_data.cc b/chrome/browser/profiles/profile_io_data.cc
-index 2f5ed4b15589ffa5bae18d7f06e6776f705f5229..62b9d86de52131eac490aeb41b55a057f4d9373e 100644
+index 2f5ed4b15589ffa5bae18d7f06e6776f705f5229..a4bd6ab7ef01d4740211eff1de086fe1493f2129 100644
 --- a/chrome/browser/profiles/profile_io_data.cc
 +++ b/chrome/browser/profiles/profile_io_data.cc
-@@ -248,6 +248,9 @@ bool ProfileIOData::IsHandledProtocol(const std::string& scheme) {
+@@ -248,6 +248,7 @@ bool ProfileIOData::IsHandledProtocol(const std::string& scheme) {
  #endif
      content::kChromeUIScheme,
      content::kChromeUIUntrustedScheme,
-+#if defined(BRAVE_CHROMIUM_BUILD)
 +    kBraveUIScheme,
```

Comment:
Unrelated to this PR, this can probably be done via a `chromium_src` override.

Reply:
I actually looked into doing that yesterday, but it was not clear to me how we would do it besides creating a define in the chromium_src override and then patching this file to include that define. It would be 1 line vs 3 lines in the patch, though, so perhaps still worth it.

@mkarolin Could you clarify what you mean?

Reply:
I think I better understood what you meant and got a PR already proposing a change, see https://github.com/brave/brave-core/pull/7995